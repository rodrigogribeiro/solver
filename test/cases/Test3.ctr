exists alpha0. typedef T as alpha0 ,
   def f : (T, int) in
      def i : T in
         exists alpha1.
	    exists alpha2.
	       exists alpha3.
	          exists alpha4. typeof(i) = alpha3, has(alpha3, x : alpha4), alpha4 = alpha1 , int = alpha2, typeof(+) = (alpha1,alpha2,int)  
