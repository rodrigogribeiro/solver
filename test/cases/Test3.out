typedef struct T {short x;} T;
